tests/cli/NewFwkLib/Utils.cs

```diff
@@ -1,1457 +0,0 @@
-/*
```

Comment:
Was this whole `NewFwkLib` directory/project unused, then?

Reply:
I think it must have been an attempt at something new a long time ago but it wasn't hooked up to anything. Should have been deleted before the initial donation.